### ISSUE TITLE
Site is down

### ISSUE THREAD

**Issue by Fred4106** [jcarbelbide/tog-crowdsourcing#10]

**Comment by Tarried** [comment]
425 right now

**Comment by mbkv** [comment]
386

**Comment by BulbagardenTech** [comment]
Over a full week and no word. Is this plugin being discontinued? We'd appreciate some kind of update.

**Comment by Justin-Folvarcik** [comment]
Can we get an update on this, please? If there's no intention of maintaining it, it needs to be removed from Runelite.

**Comment by RichardPoes** [comment]
I could set up an alternative database, but it's easier if I get a little help of the original author

**Comment by Justin-Folvarcik** [comment]
The author appears to have gone inactive, with their last activity on Github as a whole being in July.

**Comment by RichardPoes** [comment]
I have to check what kind of messages are being sent between the client and server. Seems like a fun project but my time is limited. I will comment here if I made any progression

**Comment by jcarbelbide** [comment]
Hey everyone, really sorry for missing this. Just noticed and I'll try to fix this

**Comment by jcarbelbide** [comment]
This should be resolved now. Again, extremely sorry about the really bad response time on this issue. Totally unacceptable on my part. I've fixed my notifications and this shouldn't happen again

I'm not exactly sure what happened, but the server became unresponsive and I had to bring it down and back up. I relaunched the services and it should be good again. I'm looking into better notifications so that I don't have to rely on GihHub issues to know when there's a problem
